### ISSUE TITLE
Use external VC type definitions

### ISSUE THREAD

**Issue by dmitrizagidulin** [digitalcredentials/web-verifier-plus#37]
Replace the current VC and VP type definitions in https://github.com/digitalcredentials/web-verifier-plus/blob/main/types/ with the ones exported by https://github.com/digitalcredentials/vc-data-model

**Comment by kayaelle** [comment]
@dmitrizagidulin - I'm assigning to @lautom36 - this still needs to be done?

**Comment by lautom36** [comment]
@dmitrizagidulin Is this a straight copy over or just updating? For example, verifier plus has a type OpenBadgeAchievement but vc-data-model does not. Should that type be removed?

**Comment by dmitrizagidulin** [comment]
> @dmitrizagidulin - I'm assigning to @lautom36 - this still needs to be done?

Yes, I think this still should be done. However, I think this ticket is fairly low priority, let's wait a bit on it.
